feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,363 @@
+                config:
+                    name: "nhg_B1"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_B"
+        - term:
+          config:
+            id: "match_1_dest_B2"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_B2"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_B"
+
+
+  forwarding-groups:
+    - forwarding-group: "input_dest_A"
+      config:
+        name: "input packets"
+        output-queue: dummy_input_queue_A
+    - forwarding-group: "input_dest_B"
+      config:
+        name: "input packets"
+        output-queue: dummy_input_queue_B
+
+  queues:
+    - queue:
+      config:
+        name: "dummy_input_queue_A"
+    - queue:
+      config:
+        name: "dummy_input_queue_B"
+
+  scheduler-policies:
+    - scheduler-policy:
+      config:
+        name: "limit_1Gb"
+      schedulers:
+        - scheduler:
+          config:
+            sequence: 1
+            type: ONE_RATE_TWO_COLOR
+          inputs:
+            - input: "my input policer 1Gb"
+              config:
+                id: "my input policer 1Gb"
+                # instead of QUEUE, how about a new enum, FWD_GROUP (current options are QUEUE, IN_PROFILE, OUT_PROFILE)
+                queue: dummy_input_queue_A
+          one-rate-two-color:
+            config:
+              cir: 1000000000           # 1Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+    - scheduler-policy:
+      config:
+        name: "limit_2Gb"
+      schedulers:
+        - scheduler:
+          config:
+            sequence: 1
+            type: ONE_RATE_TWO_COLOR
+          inputs:
+            - input: "my input policer 2Gb"
+              config:
+                id: "my input policer 2Gb"
+                # instead of QUEUE, how about a new enum, FWD_GROUP (current options are QUEUE, IN_PROFILE, OUT_PROFILE)
+                input-type: QUEUE
+                queue: dummy_input_queue_B
+          one-rate-two-color:
+            config:
+              cir: 2000000000           # 2Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1.100"
+        config:
+          interface-id: "PortChannel1.100"
+        input:
+          classifers:
+            - classifier:
+              config:
+                name: "dest_A"
+                type: "IPV4"
```

Comment:
I think type is not needed as policy will have terms for both IPv6 and v6.

Reply:
The OC  classifiers model is address familiy specific , so one classifier is needed for each address family.